### PR TITLE
Release 2.0.4

### PR DESCRIPTION
**Pull request by charles474**
Includes bug fixes from UAT testing.

### CODE REVIEW

File: client/src/pages/targets/TargetsListPage.test.jsx [hunk 1]
```diff
@@ -634,17 +634,13 @@ describe('TargetsListPage', () => {
...
 
-      const wrapper = mount(setMockMultiGroupContextWithTabAndTaskValues(tabData));
+    const wrapper = mount(setMockMultiGroupContextWithTabAndTaskValues(tabData));
 
-      const dropdown = wrapper.find(Select);
+    const dropdown = wrapper.find(Select);
 
-      expect(dropdown).toHaveLength(0);
-    });
+    expect(dropdown).toHaveLength(1);
```
Comment (michaelruocco) [inline client/src/pages/targets/TargetsListPage.test.jsx:644]: for this test I added

`expect(dropdown.find('select > option').at(0).text()).toEqual('RoRo V1');`

just to make it clearer which item in the list was expected to be displayed.

**Review by michaelruocco — DISMISSED** [review]
LGTM

File: client/src/pages/targets/TargetsListPage.test.jsx (hunk 1 above)
Comment (charles474) [reply client/src/pages/targets/TargetsListPage.test.jsx:644]: Ah yes, missed this considering that I pulled Shakir up on this.

I'll make a note of it and update it in the blue targeting dev branch and on the next release, it'll end up in master.

**Review by Shaks — APPROVED** [review]
lgtm

**Pull request merged**